### PR TITLE
derive PartialEq on ResolvedGitConfig

### PR DESCRIPTION
**Pull request by mzr**
I have two separate solutions that generate git config. I would like to compare that config file generated by those two is logically identical.

### CODE REVIEW

File: git-config/src/file/resolved.rs [hunk 1]
```diff
@@ -19,6 +19,7 @@ enum ResolvedTreeNode<'event> {
 /// not remember comments nor whitespace. Additionally, values are normalized
 /// upon creation, so it's not possible to retrieve the original value.
 #[allow(clippy::module_name_repetitions)]
+#[derive(PartialEq)]
```
Comment (Byron) [inline git-config/src/file/resolved.rs:22]: Is it possible to also derive `Eq`?

**Review by Byron — COMMENTED** [review]
Thanks a lot!

Please be warned that this crate will change a lot, even though it's save to use the current minor version as breakage is always indicated by semver.

File: git-config/src/file/resolved.rs (hunk 1 above)
Comment (mzr) [reply git-config/src/file/resolved.rs:22]: yup

Comment (mzr) [reply git-config/src/file/resolved.rs]: I also added Debug

**Comment by Byron** [comment]
Thanks a lot!

**Pull request merged**